scripts/05391-Tukor11.js

```diff
@@ -0,0 +1,5 @@
+const nme = "Sylvester";
+const hngid = "HNG-05391";
+console.log(
+  `Hello World, this is ${nme} with HNGi7 ID ${hngid} using JavaScript for stage 2 task`
```

Comment:
Please include your email after the word `task` and a space as highlighted [here ](https://hngi7.slack.com/archives/C014G2TMQ3G/p1591185461011700)

Reply:
the review has been done and the file has been updated as directed.
thanks i so much appreciate.